psi4/driver/wrapper_database.py

```diff
@@ -276,7 +277,7 @@ def database(name, db_name, **kwargs):
         except AttributeError:
             pass
         else:
-            if yes.match(str(database.isOS)):
+            if p4util.yes.match(str(database.isOS)):
```

Comment:
This is the bug fix line for v1.3.1?

Reply:
No, that is _a_ bug fix line.

If you look at the code that orders the RHF -> UHF switch, there's a check on `molecule.multiplicity()`. `molecule` wasn't defined until the new input file ran, before this PR. By getting rid of reap and sow, I could move the definition of molecule up, so that `molecule.multiplicity()` is defined when the code assumed it was.

This is a large enough change that I understand it not being a 1.3.1 candidate.

Reply:
thanks. yes, if the bugfix is most of this PR, I think it can wait in master.